lib/plugins/tracestorer/index.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
       case 'browsertime.chrometrace':
       case 'webpagetest.chrometrace':
         {
-          this.storageManager.writeDataForUrl(JSON.stringify(message.data, null, 0), message.name, message.url);
```

Comment:
It's supposed to warn if a Bluebird promise was created, but not returned. Anyways, good that this is fixed.

http://bluebirdjs.com/docs/warning-explanations.html#warning-a-promise-was-created-in-a-handler-but-was-not-returned-from-it